### ISSUE TITLE
Screen Reader: Workspace-Reports: The switch button is announced without part of displayed text

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76907]
## Action Performed:
Prerequisite:
- User is logged in

Steps:
1. Using Chrome, open the URL: https://new.expensify.com/
2. Tab to focus on the required workspace and press Enter to activate it
3. Once the Workspace menu is displayed, tab to focus on the "Reports" menu item and press Enter to activate it
4. Tab to focus on the "Report fields" toggle switch and press Enter to expand it
5. Observe the screen reader announcement

Other Occurrences:
1. On Workspaces - Rules
2. On Workspace - More features

## Expected Result:
The switch button should be announced with the complete visible label and description.
Expected Announcement:
Report fields, Report fields apply to all spend and can be helpful when you'd like to prompt for extra information, switch off

## Actual Result:
When navigating to the switch button, only the visible label "Report fields" is announced by the screen reader. The descriptive text displayed below the visible label (e.g., “Report fields apply to all spend and can be helpful when you'd like to prompt for extra information.") is not conveyed to the users. As a result, users may not fully understand the purpose or context of the switch.

HTML:
<div class="css-g5y9jx r-18u37iz r-1awozwy r-1wtj0ep"><div class="css-g5y9jx r-18u37iz r-1awozwy r-13awgt0"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div dir="auto" class="css-146c3p1" style="color: rgb(231, 236, 233); font-size: 17px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 700; line-height: 20px; white-space: pre-wrap; margin-bottom: 4px;">Report fields</div><div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-right: 20px; margin-top: 4px; line-height: 20px;">Report fields apply to all spend and can be helpful when you'd like to prompt for extra information.</div></div></div><div class="css-g5y9jx" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="opacity: 1;"><button aria-checked="true" aria-label="Report fields" role="switch" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-y47klf r-h0d30l r-1777fci r-1oqcu8e r-ttb5dx" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="background-color: rgb(3, 212, 124);"><div class="css-g5y9jx r-1awozwy r-18lll9h r-1mxhqmb r-18yzcnr r-1777fci r-ws9h79 r-u8s1d r-yc9v9c" data-component="AnimatedComponent" data-element="Component" data-source-file="createAnimatedComponent.js" style="transform: translateX(20px);"></div></div></button></div></div>

## Area issue was found in:
Workspace-Reports

## Failed WCAG checkpoints
WCAG 2.4.6 Headings and Labels
https://www.w3.org/WAI/WCAG22/Understanding/headings-and-labels

## User impact:
Descriptive headings are necessary for users who have disabilities that make reading slow and for people with limited short-term memory. These users benefit when section titles make it possible to predict what each section contains. Form input controls with labels that clearly describe the content that is expected to be entered helps users know how to successfully complete the form.

## Suggested resolution:
Use aria-labelledby attribute to programmatically associate the full visible label and relevant description with the switch control so that the entire displayed text is announced accurately when the user navigates to it.

Refer:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-labelledby
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-describedby

## Environment(s)
Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1918" height="972" alt="Image" src="https://github.com/user-attachments/assets/c8ab8346-1dbb-4fa1-bee1-c00e18bd5b5e" />

https://github.com/user-attachments/assets/b0d9bc9d-4db8-4857-9bd3-30db2e4b0319

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6996233
**Issue reported by:** NA
**Slack conversation:** NA
